cognite/neat/app/api/utils/logging.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+
+class EndpointFilter(logging.Filter):
```

Comment:
Shouldn't this ideally be controlled by log level? That is instead of just blocking endpoints from logging, we use DEBUG, INFO, WARNINGS, ERROR, etc.